UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs

```diff
@@ -50,49 +53,59 @@ public void RunOreGenerator()
 		wallTilemap = metaTileMap.Layers[LayerType.Walls].GetComponent<Tilemap>();
 		tileChangeManager = GetComponent<TileChangeManager>();
 
-		if (CustomNetworkManager.IsServer)
+		if (CustomNetworkManager.IsServer) return;
```

Comment:
Are you sure you want to return if it's the server?
```suggestion
		if (CustomNetworkManager.IsServer == false) return;
```

Reply:
Good catch